build/generate/featureMatrix.go

```diff
@@ -92,38 +110,58 @@ func matrixData() *FeatureMatrix {
 		DomainModifierDhcid  = "[`DHCID`](../language-reference/domain-modifiers/DHCID.md)"
 		DomainModifierDname  = "[`DNAME`](../language-reference/domain-modifiers/DNAME.md)"
 		DomainModifierDnskey = "[`DNSKEY`](../language-reference/domain-modifiers/DNSKEY.md)"
-		DualHost             = "dual host"
+		DualHost             = "[dual host](../dual-host.md)"
```

Comment:
Hi @chicks-net, thanks for the contribution!

Just a quick heads-up:
* The links/paths in the Markdown files currently point to GitHub URLs. This is likely because the new documents (concurrency-verified.md and dual-host.md) haven’t yet been added to the SUMMARY.md file.
    * https://docs.dnscontrol.org/provider/index#provider-api
* Once they’re included in SUMMARY.md, they’ll be part of the DNSControl documentation site structure, and the links can (and should) reference the documentation site instead.

Let me know if you need any help updating this!

Reply:
#3607 is my ticket for this.